src/cmd/descriptor.rs

```diff
@@ -57,12 +57,16 @@ pub struct CountEdges {
     /// The input descriptor file to generate signing bytes for
     #[arg(default_value = "descriptor.json")]
     input: PathBuf,
-    /// The file to write the resulting signing bytes to
-    #[arg(default_value = "edgecount.csv")]
+    /// The file to write the resulting edgecounts to
+    #[arg(default_value = "edgecount.json")]
     output: PathBuf,
 }
 
 impl CountEdges {
+    pub fn new(input: PathBuf, output: PathBuf) -> Self {
```

Comment:
Why do you need a new fucntion?

Reply:
ah yes, I've removed it [1dc4739](https://github.com/helium/xorf-generator/pull/30/commits/1dc473963b5da1ad92afa1d76811f83a2072bda8).

Also, I realized that I can do this all locally separately anyway. So your call if you want to merge this or not.